client/stylesheets/_toCat.scss

```diff
@@ -4850,13 +4862,23 @@ On click the images will be displayed at normal size to complete the effect
 	left:12px;
 }
 
+.optionsItems li a:hover {
+	background-color: #d3d3d3 !important;
+}
+
+.optionsItems li a i:hover {
+	color: #fff !important;
+}
+
+.optionsItems li a i {
+	margin-right: 12px;
+}
+
 .optionsClick ul li a {
-	transition:background-color 0.2s ease-in-out;
-	-webkit-transition:background-color 0.2s ease-in-out;
-	-moz-transition: background-color 0.2s ease-in-out;
 	display:block;
-	padding:8px 10px 8px 40px;
-	background:#FFF;color:#333;
+	padding: 15px;
+	background:#fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented with spaces, not tabs<br>Colon after property should be followed by one space